src/config/configuration.ts

```
@@ -254,7 +254,10 @@ export default () => {
                 baseUrl: process.env.FINEARTS_URL,
                 queryUrl: process.env.FINEARTS_QUERY_URL,
             },
-            // 디자인융합학과 (마지막 페이지:16)
+            INHADESIGN: {
```

ESLint

[error] Delete `······`

(prettier/prettier)